packages/app/src/systems/Sidebar/components/Menu/Menu.tsx

```diff
@@ -42,7 +43,10 @@ function MenuItemContent({ item, isOpened }: MenuItemContentProps) {
         css={styles.menuItemContent(Boolean(isOpened))}
         animate={{ height: isOpened ? '100%' : '24px' }}
       >
-        <Flex gap="$3">
+        <Flex
+          css={pathname === item.path ? styles.activeRoute : styles.route}
```

Comment:
This maybe will imply a bug if the pathname includes the path but it isn't the path strictly.
You can use the `useResolvePath` and `useMatch` instead to generate this behavior (like ReactRouter [recommends here](https://github.com/remix-run/react-router/blob/main/packages/react-router-dom/index.tsx#L440-L441))

```jsx
const location = useLocation();
const path = useResolvedPath(location);
const match = useMatch({ path: path.pathname, end, caseSensitive });
```